package.json

```diff
@@ -2,15 +2,16 @@
   "name": "@orbs-network/orbs-ethereum-contracts-v2",
   "version": "0.0.31",
   "description": "",
-  "main": "./release/test/test-kit.js",
+  "main": "./release/index.js",
+  "types": "./release/index.d.ts",
   "repository": "https://github.com/orbs-network/orbs-ethereum-contracts-v2",
   "scripts": {
     "test": "export GANACHE_CORE=true && npm run prepublish && mocha --exit --full-trace --async-stack-traces -r ./tsinit.js --exclude ./test/gas-scenarios.spec.ts ./test/*.spec.ts --timeout 1200000000",
     "test-ropsten": "export ETHEREUM_URL=https://ropsten.infura.io/v3/62f4815d28674debbe4703c5eb9d413c && export WEB3_DRIVER_VERBOSE=true && npm run prepublish && mocha --bail --exit --full-trace -r ./tsinit.js --exclude ./test/gas-scenarios.spec.ts ./test/*.spec.ts --timeout 1200000000",
     "test-kovan": "export ETHEREUM_URL=https://kovan.infura.io/v3/62f4815d28674debbe4703c5eb9d413c && export WEB3_DRIVER_VERBOSE=true && npm run prepublish && mocha --bail --exit --full-trace -r ./tsinit.js --exclude ./test/gas-scenarios.spec.ts ./test/*.spec.ts --timeout 1200000000",
     "test-gas-scenarios": "export GANACHE_CORE=true && npm run prepublish && mocha --exit --full-trace -r ./tsinit.js ./test/gas-scenarios.spec.ts --timeout 600000",
     "coverage": "export ETHEREUM_URL=http://localhost:8555 && truffle run coverage",
-    "prepublish": "echo 'Compiling .sol and .ts...' && rm -rf ./release/ && tsc --project tsconfig.json --declaration --declarationMap --skipLibCheck --outDir ./release && truffle compile && mkdir -p release/build/contracts release/typings && cp -r build/contracts/. release/build/contracts && cp -r typings/. release/typings && ./extractAbi.sh",
+    "prepublish": "echo 'Compiling .sol and .ts...' && rm -rf ./release/ && rm -rf ./build/ && rm -rf ./abi/ && tsc --project tsconfig.json --declaration --declarationMap --skipLibCheck --outDir ./release && truffle compile && mkdir -p release/build/contracts release/typings release/abi &&  cp -r build/contracts/. release/build/contracts && ./extractAbi.sh && cp -r abi/. release/abi",
```

Comment:
if we were using github issues i'd open one to extract this long script to a bash script or otherwise clean this up

Reply:
agree, and we are :)